hazelcast/src/main/java/com/hazelcast/map/impl/operation/TryRemoveOperation.java

```diff
@@ -20,10 +20,11 @@
 import com.hazelcast.nio.ObjectDataInput;
 import com.hazelcast.nio.ObjectDataOutput;
 import com.hazelcast.nio.serialization.Data;
+import com.hazelcast.spi.impl.MutatingOperation;
 
 import java.io.IOException;
 
-public class TryRemoveOperation extends BaseRemoveOperation {
+public class TryRemoveOperation extends BaseRemoveOperation implements MutatingOperation {
```

Comment:
Since `BaseRemoveOperation` already `implements MutatingOperation`, no reason to duplicate it here. Same goes for `RemoveIfSame`, `Remove`, `TxnDelete` and `Delete` operations.

Reply:
Thx. Fixed.